records_mover/records/hints/conversions.py

```diff
@@ -0,0 +1,62 @@
+from .hint import LiteralHint, StringHint
+import chardet
+from .types import (
+    RecordsHints, BootstrappingRecordsHints, HintHeaderRow, HintCompression, HintQuoting,
+    HintDoublequote, HintEscape, HintEncoding, HintDateFormat, HintTimeOnlyFormat,
+    HintDateTimeFormatTz, HintDateTimeFormat
+)
+from .csv_streamer import stream_csv, python_encoding_from_hint
+import io
+import logging
+from .types import MutableRecordsHints
+from typing import Iterable, List, IO, Optional, Dict, Union, TYPE_CHECKING
+from typing_extensions import Literal
+if TYPE_CHECKING:
+    from pandas.io.parsers import TextFileReader
+
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: after merging sniffing improvements, move to a new file
+# TODO: this shouldn't have to be a union
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker